tests/fixtures/antipatterns/label-line-height.html

```
@@ -0,0 +1,89 @@
+      <div class="case">
+        <span class="case-label">Font size above 13px</span>
+        <div role="button" class="pass-large-font">"Font size above 13px"</div>
+      </div>
+      
+      <div class="case">
+        <span class="case-label">Not a UI label</span>
+        <p class="pass-paragraph">"Not a UI label"</p>
+      </div>
+      
+      <div class="case">
+        <span class="case-label">Badge ratio <= 1.7</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)